molecule/release-static/group_vars/all

```diff
@@ -18,4 +18,3 @@ pulp_install_plugins:
 pulp_settings:
   secret_key: secret
   content_origin: "https://{{ ansible_fqdn }}"
-pulp_webserver_server: apache
```

Comment:
why dropping it?

Reply:
whoops, that was leftover from me testing with molecule